source/common/api/win32/os_sys_calls_impl.cc

```diff
@@ -14,14 +14,15 @@ namespace Envoy {
 namespace Api {
 namespace {
 
-using WSABUFPtr = std::unique_ptr<WSABUF[]>;
+using WSAMSGPtr = std::unique_ptr<WSAMSG>;
 
 struct wsabufResult {
-  DWORD num_vec_;
-  WSABUFPtr wsabuf_;
+  WSAMSGPtr wsabuf_;
```

Comment:
WSAMSGPtr wsamsg_

Reply:
Fixed in 0b477d9efcbf75b5a193b70ef7e441a33b8afb85